playbooks/byo/generate_prometheus.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
@jcantrill I didn't know where to put this file in the tree. I just use it to run the rule. It might be obsolete in the future?